utils/index.d.ts

```diff
@@ -880,6 +880,55 @@ export interface IWizardOptions<T extends IActionContext> {
     showLoadingPrompt?: boolean;
 }
 
+export interface ActivityTreeItemOptions {
+    label: string;
+    contextValuePostfix?: string;
+    collapsibleState?: TreeItemCollapsibleState;
+    children?: (parent: AzExtParentTreeItem) => AzExtTreeItem[];
```

Comment:
I wonder if it should be `AzExtTreeItem[] | Promise<AzExtTreeItem[]>`. There might be a case where they need to do something async.